store/migration/10002__1.1.0.sql

```diff
@@ -10,6 +10,7 @@ ALTER TABLE vcs ADD CONSTRAINT vcs_type_check CHECK (type IN ('GITLAB_SELF_HOST'
 ALTER TABLE sheet ADD source TEXT NOT NULL CHECK (source IN ('BYTEBASE', 'GITLAB_SELF_HOST', 'GITHUB_COM')) DEFAULT 'BYTEBASE';
 ALTER TABLE sheet ADD type TEXT NOT NULL CHECK (type IN ('SQL')) DEFAULT 'SQL';
 ALTER TABLE sheet ADD payload JSONB NOT NULL DEFAULT '{}';
+CREATE UNIQUE INDEX idx_sheet_project_id_source_name ON sheet(project_id, source, name);
```

Comment:
Not sure if we need to enforce this. Otherwise 2 different users can't create a sheet with the same name.

Reply:
And they may not know each other if the sheet is private

Reply:
True. On the other hand, when sheets are public, two sheets with the same name might be confusing if there's no other identifier. Trying to think of a better way to handle this.

Reply:
How's the visibility determined if the sheet is sourced from VCS - default to PROJECT?

Reply:
> True. On the other hand, when sheets are public, two sheets with the same name might be confusing if there's no other identifier. Trying to think of a better way to handle this.

Having the same name is always confusing.  On the other hand, sheet is more like docs and we may have a similar dashboard. It's just too strict to enforce the name uniqueness (and in general name uniqueness is not enforced, id is)

Reply:
PTAL, updated: 
```sql
CREATE INDEX idx_sheet_project_id ON sheet(project_id);
CREATE INDEX idx_sheet_name ON sheet(name);
```